ad7768-1fmcz/ad77681.c

```diff
@@ -287,6 +316,287 @@ int32_t ad77681_spi_read_adc_data(struct ad77681_dev *dev,
 	return ret;
 }
 
+/**
+ * This function uses continuous read mode, which has to be enabled in advance
+ * Read conversion result from device using GPIO DRDY interrupt in 16bit frames
+ * Firtst 24bits or 16bits of data	dev->conv_length   followed by
+ * 8bits of status bit, if enabled	dev->sataus_bit    followed by
+ * 8bits of CRC, if enabled			dev->crc_sel
+ *
+ * @param dev - The device structure.
+ * @param measured_data - structure carying ADC measurement info
+ * @return 0 in case of success, negative error code otherwise.
+ */
+/*
+*	    /-------------\       /-------------\       /-------------\
+*------  24-bit data  -------  8-bit staus  -------   8-bit CRC   -------
```

Comment:
I would not use this type of comments (sequence drawing with characters). The doxygen build will most probably mess it up. Maybe you can reference this sequence from the datasheet.

Reply:
done removing thin comment